ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/CliqueGenesisFileModule.java

```diff
@@ -18,13 +18,18 @@
 import org.hyperledger.besu.config.GenesisConfigOptions;
 import org.hyperledger.besu.consensus.clique.CliqueProtocolSchedule;
 import org.hyperledger.besu.consensus.ibft.IbftBlockHeaderFunctions;
+import org.hyperledger.besu.crypto.KeyPairSecurityModule;
+import org.hyperledger.besu.crypto.NodeKey;
+import org.hyperledger.besu.crypto.SECP256K1;
 import org.hyperledger.besu.ethereum.core.BlockHeaderFunctions;
 import org.hyperledger.besu.ethereum.mainnet.ProtocolSchedule;
 
 import javax.inject.Named;
 
 class CliqueGenesisFileModule extends GenesisFileModule {
 
+  NodeKey nodeKey = new NodeKey(new KeyPairSecurityModule(SECP256K1.KeyPair.generate()));
```

Comment:
In my opinion I think it is possible to add `private final` here

Reply:
fixed.